authentication-demo-ui/pom.xml

```diff
@@ -5,45 +5,62 @@
 	<parent>
     <groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.0.2.RELEASE</version>
+		<version>3.2.5</version>
```

Comment:
You can get this version from kernel-bim, no need to specify

Reply:
https://github.com/mosip/commons/blob/develop-java21/kernel/kernel-bom/pom.xml

But there is no such dependency or version here